lib/stdlib/test/erl_lint_SUITE.erl

```diff
@@ -4448,6 +4449,46 @@ inline_nifs(Config) ->
            {warnings,[{2,erl_lint,nif_inline}]}}],
     [] = run(Config, Ts).
 
+compile_options(Config) ->
+    WarnMissSpecOkay
+        = {warn_missing_spec_ok,
+           <<"-export([external/0]).
+              -spec external() -> ok. external() -> internal().
+              internal() -> ok.">>,
+           [warn_missing_spec],
+           []},
+
+    WarnMissSpecNotOkay
+        = {warn_missing_spec_error,
+           <<"-export([external/0]).
+              external() -> internal().
+              -spec internal() -> ok. internal() -> ok.">>,
+           [warn_missing_spec],
+           {warnings, [{2, erl_lint, {missing_spec, {external, 0}}}]}},
```

Comment:
You can combine the two tests.

You are not testing that there will **not** be a warning for a local function without a spec.

Here is a suggestion (not tested):

```suggestion
    WarnMissSpec =
        {warn_missing_spec,
           <<"-export([external_with_spec/0,external_without_spec/0]).
              external_without_spec() -> internal_without_spec().
              spec externa_with_specl() -> ok.
              external_with_spec() -> internal_with_spec().
              -spec internal_with_spec() -> ok.
              internal_with_spec() -> ok.
              internal_without_spec() -> ok.>>,
           [warn_missing_spec],
           {warnings, [{2, erl_lint, {missing_spec, {external_without_spec, 0}}}]}},
```

Reply:
Hm... I want to test the following:
1. no error for `warn_missing_spec` when everything's OK,
2. error for `warn_missing_spec` when there's something missing.

Reply:
In the second test (`WarnMissSpecNotOkay`), I'm making sure that `warn_missing_spec` doesn't care about internal specs.